packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -15,10 +15,29 @@ export default class AppStore {
         this.api_helpers_store = null;
     }
 
+    error_for_non_eu_client = {
```

Comment:
Please make this a function. If the language is changed, this object will remain the same containing old translation text.

Reply:
Alright. Done

Reply:
fixed✅